Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -4131,3 +4131,118 @@ WebKitFeatureList* webkit_settings_get_development_features(void)
 {
     return webkitFeatureListCreate(WebPreferences::internalDebugFeatures());
 }
+
+/**
+ * webkit_settings_apply_from_key_file:
+ * @settings: a #WebKitSettings
+ * @key_file: a #GKeyFile
+ * @group_name: Name of the group to read from @key_file
+ * @error: return location for error or %NULL to ignore
+ *
+ * Reads the contents of the given @group_name from the given @key_file and apply the value of
+ * each key/value to the corresponding property on the @settings.
+ *
+ * Value types have to match with the corresponding setting property type and it is not allowed and
+ * the group keys have to match existing setting property names. If those conditions are not met,
+ * the function will return %FALSE.
+ *
+ * Supported value types are strings, booleans and unsigned integers.
+ *
+ * Returns: %TRUE if the settings were correctly applied or %FALSE otherwise.
```

Comment:
```suggestion
 * Returns: %TRUE if the settings were correctly applied or %FALSE on error.
```

To avoid any doubt that `FALSE` corresponds to `error` being set.